jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java

```diff
@@ -219,7 +219,8 @@ private void executionQueue(Execution message) {
                 workerTasksDedup
                     .stream()
                     .filter(workerTask -> workerTask.getTask().isSendToWorkerTask())
-                    .forEach(workerTaskQueue::emit);
+                    .forEach(workerTask -> workerTaskQueue.emit(
+                        workerTask.getTask().getWorkerGroup() != null ? workerTask.getTask().getWorkerGroup().getKey() : null, workerTask));
```

Comment:
You should have a service that will map the task to a proper worker group (to support advanced labels in the future)

Reply:
It was planned for the next feature on worker group but I can do it now if you want.

Reply:
Done